Source/API/Internal.lua

```diff
@@ -0,0 +1,13 @@
+function Auctionator.API.ComposeError(callerID, message)
+  error(
+    "Contact the maintainer of " .. callerID ..
+    " to resolve this problem. Details: " .. message
+  )
+end
+
+-- TODO: Maintain authorization keys for add-ons to prevent false callerIDs
```

Comment:
Note for ourselves, we should probably store encrypted values and encrypt the provided string so people can't just use other addon's names, if we go this route

Reply:
Yeah, for now its just a "fear factor", but it is something for the future